dev-build/conf/routes

```diff
@@ -92,6 +92,7 @@ GET            /most-read-facebook.json
 GET            /most-read-twitter.json                                                                                           controllers.MostViewedSocialController.renderMostViewed(socialContext: String = "twitter")
 GET            /most-read.json                                                                                                   controllers.MostPopularController.render(path = "")
 GET            /most-read/*path.json                                                                                             controllers.MostPopularController.render(path)
+GET            /most-read-amp.json                                                                                               controllers.MostPopularController.renderPopularAmp()
```

Comment:
Yes that makes sense. I will rename. in this case though it would be `/most-read-mf2.json`. Doesn't exactly roll off the tongue :wink: